frontend/.eslintrc.json

```diff
@@ -8,6 +8,7 @@
     "@vue/typescript"
   ],
   "rules": {
+    "vue/multi-word-component-names": "off",
```

Comment:
May I know what is this for and why is it needed for this PR?

Reply:
Oh sorry I forgot to remove this, it is to allow the naming of one-word component like "ramp", it was added previously when I try to remove the "-v", but later when I added the "-c", this setting is useless.  I have removed it now.

Reply:
Thank you for the clarification!